content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -192,7 +192,7 @@ For more information on version skews, see:
 cat <<EOF | sudo tee /etc/yum.repos.d/kubernetes.repo
 [kubernetes]
 name=Kubernetes
-baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch
```

Comment:
The original text looks correct to me. The `\` tells your POSIX-style shell not to interpret it as a shell variable within the heredoc.

Have I misunderstood how the shell command works @fossilet?

Reply:
Aha,  I copied and pasted to the repo file. My fault.